test/python/WMCore_t/WorkQueue_t/LocalQueueProfile_t.py

```diff
@@ -14,7 +14,7 @@
 from WMQuality.Emulators.DataBlockGenerator import Globals
 from WMQuality.Emulators.WMSpecGenerator.WMSpecGenerator import WMSpecGenerator
 from WMCore.WorkQueue.WorkQueue import localQueue
-from .WorkQueueTestCase import WorkQueueTestCase
+from WorkQueueTestCase import WorkQueueTestCase
```

Comment:
This is not correct python3. You need to add WMCore_t.WorkQueue_t. in front. In general, relative imports are bad, especially if we don't need them.